fs/zbd_zenfs.cc

```diff
@@ -398,8 +399,8 @@ ZonedBlockDevice::ZonedBlockDevice(std::string bdevname, ZbdBackendType backend,
     zbd_be_ = new ZbdlibBackend(bdevname);
     Info(logger_, "New Zoned Block Device: %s", zbd_be_->GetFilename().c_str());
   } else if (backend == ZbdBackendType::kZoneFS) {
-    zbd_be_ = NULL;
-    Info(logger_, "Unsupported zonefs backing requested");
+    zbd_be_ = new ZoneFsBackend(bdevname);
+    Info(logger_, "New zonefs backing: %s", zbd_be_->GetFilename().c_str());
   }
```

Comment:
> Maybe add an else which logs "Unsupported ZBD backend requested" and initializes `zbd_be_ = NULL`.

It would be dead code because the match on the enum is exhaustive. Probably the compiler would remove it :shrug: